x-pack/legacy/plugins/infra/public/components/logging/log_analysis_setup/user_management_link.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButton, EuiButtonProps } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React from 'react';
+
+export const UserManagementLink: React.FunctionComponent<EuiButtonProps> = props => (
```

Comment:
Should this render if the user doesn't have permissions to manage the users?

<img width="1093" alt="Screenshot 2020-01-29 at 12 24 06" src="https://user-images.githubusercontent.com/57448/73352828-4647f200-4292-11ea-9928-053f9a0c60ed.png">

Reply:
Well, the target page exists and gives actionable advice. :thinking: Is that too bad of a UX? @katrin-freihofner what do you think?

Reply:
First of all, I'm confused about the `machine_learning_user` role - why do we need this role? Second, I think the message proposed in this PR is good. Where does the button `Manage users` link to?

Reply:
The role is required to be able to check the status of ML jobs and to access their results.

The button links to the user management section in Kibana. That is what @afgomez  took the screenshot of above.

Reply:
Before this PR we required the `machine_learning_admin` role even to read the results, which made it barely usable for sensible Elasticsearch deployments.

Reply:
Ok, I understand. One more question: does it make sense that we take the user to the user management screen? Are there any actions they can take? If not, I'd rather display this in the `Additional ML privileges required` message and remove the button.

Reply:
That screen allows the user to assign the role if they have the permission. I thought a call to action would be preferable. I can remove the button, though.

Reply:
I'll merge this fix for now to get into a BC. We can tweak the UX in follow-up PRs.